Client/WarFare/GameProcMain.cpp

```diff
@@ -4732,6 +4747,9 @@ bool CGameProcMain::CommandToggleUIInventory()
 	{
 		bNeedOpen = true;
 		if(m_pUISkillTreeDlg->IsVisible()) m_pUISkillTreeDlg->Close();
+
+		if (m_pUILevelGuide->IsVisible())
```

Comment:
All of this these visibility updates are unnecessary. Add the style UISTYLE_POS_RIGHT and it'll be mutually exclusive with them (this is the official behaviour).